app/src/main/AndroidManifest.xml

```diff
@@ -50,6 +50,7 @@
             android:theme="@style/AppTheme.NoActionBar" />
         <activity
             android:name=".activities.BluetoothConnectionActivity"
+            android:launchMode = "singleInstance"
```

Comment:
According to this site it's not a good idea to use that: https://inthecheesefactory.com/blog/understand-android-activity-launchmode/en

> This mode is rarely used. Some of the real use case is an Activity for Launcher or the application that you are 100% sure there is only one Activity. Anyway I suggest you not to use this mode unless it is really necessary.